pages/common/ollama.md

```diff
@@ -19,6 +19,14 @@
 
 `ollama list`
 
+- Update pulled models (Referenced from <https://github.com/ollama/ollama/issues/1890#issuecomment-1885381713>):
+
+`ollama list | tail -n +2 | awk '{print $1}' | while read -r model; do  sudo ollama pull $model; done`
```

Comment:
```suggestion
- Pull/Update a specific model:

`ollama pull {{model}}`
```

Since this is a cross-platform command, I wouldn't suggest documenting the latter atm, instead, I would suggest looking out for PRs/commits for an official command/flag for this i.e. https://github.com/ollama/ollama/pull/2179.

For now, we can have an example which works for both pulling and updating specific models.

Reply:
You're right, it's coming sometime in the future but not quite there yet. I think that command is useful for Linux/Unix users so I included that line. Do you think we can include the command specifying that it is for Linux/Unix, and update the command as soon as the official command is implemented? There are some commands in tldr that targets specific application (such as history, you have bash/zsh specific commands), so I thought it's fine to do so as long as you specify.

Reply:
> You're right, it's coming sometime in the future but not quite there yet. I think that command is useful for Linux/Unix users so I included that line. Do you think we can include the command specifying that it is for Linux/Unix, and update the command as soon as the official command is implemented? There are some commands in tldr that targets specific application (such as history, you have bash/zsh specific commands), so I thought it's fine to do so as long as you specify.

Maybe, we can add a separate page under the `linux` directory i.e. at `pages/linux/ollama.md` where in addition to the common commands we can document the Linux specific commands too and have the current page document the cross-platform commands only (i.e. keeping `olama pull {{model}}` instead of the specific command and removing Linux commands); we follow a similar approach for UNIX commands where POSIX variants are documented in the `common` directory and platform-specific command are documented in separate duplicate pages.

Note: The clients will display the platform-specific pages of your respective platform over the page in `common` if it exists.

Reply:
I think you're right, we can wait until the discussed command is implemented in Ollama, and then we include that command in `common/ollama.md`. I think creating a duplicate (except for one command) under a `linux` directory makes it a little harder to maintain. I think we can go with your suggestion, thanks @kbdharun !